engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/instrument/RuntimeAsyncCommandsTest.scala

```diff
@@ -242,4 +243,93 @@ class RuntimeAsyncCommandsTest
     diagnostic.message shouldEqual Some("sleep interrupted")
     diagnostic.stack should not be empty
   }
+
+  it should "interrupt running execution context and recompute" in {
```

Comment:
I'd rename the test saying that there are no `Panic` updates when the program is interrupted. Because there are similar interruption tests in the suite already